source/unified-test-format/unified-test-format.rst

```diff
@@ -1798,6 +1857,155 @@ Use a ``listIndexes`` command to check whether the index exists. Note that it is
 currently not possible to run ``listIndexes`` from within a transaction.
 
 
+loop
+~~~~
+
+The ``loop`` operation executes sub-operations in a loop.
+It supports the following arguments:
+
+- ``operations``: the sub-operations to run on each loop iteration.
+  Each sub-operation must be a valid operation as described in this
+  specification.
```

Comment:
Can we link to https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#entity-test-operations here to reduce ambiguity?

Reply:
Added.